### PR TITLE
report generation

### PR DESCRIPTION
**Pull request by alorenzo175**
Super rough report generation building on #88. Will try and get it somewhat tested and reduce hackiness, but will likely have to come back and improve dramatically.

### CODE REVIEW

**Comment by alorenzo175** [comment]
@wholmgren I don't know if you want to take a look at this. I think it is ready enough to merge. I think we should get this in by ~tomorrow so the api and dashboard can use it.

**Comment by wholmgren** [comment]
I’ll try to review tonight.

File: solarforecastarbiter/io/utils.py [hunk 1]
```diff
@@ -213,6 +221,35 @@ def adjust_timeseries_for_interval_label(data, interval_label, start, end):
     return data.loc[start:end]
 
 
+def serialize_data(values):
+    serialized_buf = pa.serialize(values).to_buffer()
```
[the remark this answers is not in the document]
Comment (alorenzo175) [reply solarforecastarbiter/io/utils.py]: Not too special. The report and processed data needs to be serialized to be sent to the API and stored in MySQL. Arrow nicely handles pandas objects and is faster to de/serialize and more stable (probably) than msgpack.

File: solarforecastarbiter/reports/main.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
+"""
+Make a report.
+
+Steps:
```
[the remark this answers is not in the document]
Comment (alorenzo175) [reply solarforecastarbiter/reports/main.py]: still accurate; I've updated them a bit

**Pull request merged**